infra/terraform/modules/user_nfs_softnas/main.tf

```diff
@@ -0,0 +1,31 @@
+variable "env" {}
+variable "vpc_id" {}
+variable "dns_zone_id" {}
+variable "dns_zone_domain" {}
+variable "node_security_group_id" {}
+variable "bastion_security_group_id" {}
+variable "ssh_public_key" {}
+
+variable "subnet_ids" {
+    type = "list"
+}
+
+variable "num_instances" {
+    default = 2
+}
+
+variable "instance_type" {
+    default = "m4.large"
+}
+
+variable "nfs_mountpoint_ip" {
+    default = "172.16.0.1"
```

Comment:
Yes - it's an IP outside of the IP range we use elsewhere, so it won't collide with anything. This is how SoftNAS handles failover - it adds a rule to the routing table to direct traffic for that IP to the current active SoftNAS primary - when failover happens and the secondary takes over it updates the routing table to send traffic for that IP to itself.